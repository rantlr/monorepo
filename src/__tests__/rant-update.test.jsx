import * as React from 'react';
import { render, cleanup } from 'react-testing-library';
import RantUpdate from '../rant-update';

const createdAt = '2018-11-17T15:19:52.675Z';
const updatedAt = '2018-12-23T10:11:12.131Z';

afterEach(cleanup);

test('snapshot', () => {
  const rant = {
    id: '123',
  };

  const rantUpdate = {
    id: '1234',
    body: 'some body',
    created_at: createdAt,
    updated_at: updatedAt,
  };

  render(
    <RantUpdate rant={rant} rantUpdate={rantUpdate} onDelete={jest.fn()} />,
  );

  expect(document.documentElement).toMatchSnapshot();
});
