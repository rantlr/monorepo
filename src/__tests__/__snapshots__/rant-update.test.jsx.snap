// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<html>
  <head>
    <style
      data-emotion=""
    >
      
    </style>
    <style
      data-emotion=""
    >
      
      .css-hqfkvb{border:solid hotpink 1px;padding:2px;}
    </style>
    <style
      data-emotion=""
    >
      
      .css-1e0neex{margin-top:8px;}
    </style>
  </head>
  <body>
    <div>
      <article
        class="css-hqfkvb"
        id="rant-update-1234"
      >
        <p>
          some body
        </p>
        <button
          class="css-1e0neex"
          type="button"
        >
          Edit
        </button>
        <button
          class="css-1e0neex"
          type="button"
        >
          Delete
        </button>
        <p>
          <a
            href="#rant-update-1234"
          >
            Created 
            16:19:52 on Saturday November 17th 2018
          </a>
        </p>
        <p>
          Updated 
          11:11:12 on Sunday December 23th 2018
        </p>
      </article>
    </div>
  </body>
</html>
`;
